.github/workflows/update-wpt.yml

```diff
@@ -0,0 +1,70 @@
+name: WPT update
+
+on:
+  schedule:
+    # Run once a week at 12:00 AM UTC on Sunday.
+    - cron: 0 0 * * *
+  workflow_dispatch:
+
+permissions:
+  contents: read
+
+jobs:
+  wpt-subsystem-update:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        subsystem:
```

Comment:
Can you make the subsystem be an argument that's able to be passed to workflow_dispatch? That way we can trigger this for specific subsystem(s) that aren't necessarily run in a cron as needed too.

https://github.com/nodejs/node/blob/4f1fe8a01553ecfe8e63a484165313fe3fd8c10e/.github/workflows/daily-wpt-fyi.yml#L9-L13

https://github.com/nodejs/node/blob/4f1fe8a01553ecfe8e63a484165313fe3fd8c10e/.github/workflows/daily-wpt-fyi.yml#L29-L32

Reply:
applied thank you